src/features/nav/getNavData.ts

```diff
@@ -94,6 +99,14 @@ export const useNavQuery = () => {
       return { ...data, profile };
     },
     {
+      onSettled: async (_, error: any) => {
+        if (isErrorWarrantingLogout(error)) {
+          // eslint-disable-next-line no-console
+          console.log('logging out due to auth error in useNavQuery');
+          await logout();
```

Comment:
the logout action will fail since we dont have an valid auth token... does it fail silently and still do the other browser side logout stuff? seems maybe we want to use a different hook that won't make a client.mutate which will fail? @CryptoGraffe